cpp/include/cudf/aggregation.hpp

```diff
@@ -43,6 +43,20 @@ namespace detail {
 class simple_aggregations_collector;
 class aggregation_finalizer;
 }  // namespace detail
+
+/**
+ * @brief Tie-breaker method to use for ranking the column.
+ *
```

Comment:
Should we add an `@see` here to point to the excellent documentation in `cudf::make_rank_aggregation()`? The differences between these options might not be immediately obvious to the lay reader.